src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -20,24 +20,29 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
 
 {% multicode %}
   ```client-web
-  import { Client, Storage } from "appwrite";
+  import { Client, Storage, ID } from 'appwrite';
 
   const client = new Client()
-      .setEndpoint('https://cloud.appwrite.io/v1')
       .setProject('<PROJECT_ID>');
 
   const storage = new Storage(client);
 
   const promise = storage.createFile(
-      '[BUCKET_ID]',
-      ID.unique(),
-      document.getElementById('uploader').files[0]
+    "<BUCKET_ID>",
+    ID.unique(),
+    {
+      name: "file.txt",
+      type: "text/plain",
+      size: 37967,
+      uri: "file://text.png",
```

Comment:
The rest of the code sample has 4 spaces for the indentation, so this should be indented with 4 spaces too instead of 2:

```suggestion
        name: "file.txt",
        type: "text/plain",
        size: 37967,
        uri: "file://text.png",
```